standards/github-actions.md

```diff
@@ -1,3 +1,60 @@
+
+The following are good examples of Github Action scripts that build a Docker image, push it up to the appropriate ECR repository, and then deploy it to the appropriate ECS cluster.
+
+- [patron-info-poller](https://github.com/NYPL/patron-info-poller/blob/main/.github/workflows/deploy-qa.yml)
+- [drb-etl-pipeline](https://github.com/NYPL/drb-etl-pipeline/blob/main/.github/workflows/build-production.yaml)
+- [shared-ruby](https://github.com/NYPL/shared-ruby/blob/develop/.github/workflows/build-qa.yml)
+
+While the above examples can serve as a template, there are a few things to note:
+- Make sure to have the following before the `jobs` step which is needed for OIDC:
+
+```yml
+permissions:
+  id-token: write
+  contents: read
+```
+
+- Every AWS account has a different IAM key. Make sure to update the `role-to-assume` configuration with the proper key obtained from DevOps.
+
+```yml
+- name: Configure AWS credentials
+  uses: aws-actions/configure-aws-credentials@v2
+  with:
+    role-to-assume: arn:aws:iam::[REPLACE-THIS]:role/GithubActionsDeployerRole
+    aws-region: us-east-1
+```
+
+- In the build step, make sure to update the `ECR_REPOSITORY` with the app's ERC repo name.
+- In the `aws ecs update-service` command step, update the app's ECS cluster and service name.
+
+## Tests and other actions
+
+Github Actions are also great for runnign unit tests, creating git tags, publishing to npm, or running any other automated tasks.
+
+Examples of Actions that run unit tests:
+- [patron-info-poller](https://github.com/NYPL/patron-info-poller/blob/main/.github/workflows/run-tests.yml) (python)
+- [circulation](https://github.com/NYPL-Simplified/circulation/blob/main/.github/workflows/test.yml) (python)
+- [research-catalog](https://github.com/NYPL/research-catalog/blob/main/.github/workflows/testing.yml) (node)
+- [digital-collections](https://github.com/NYPL/digital-collections/blob/main/.github/workflows/ci.yml) (node)
+
+Other actions include but are not limited to:
+- [deploying to Github pages](https://github.com/NYPL/nypl-design-system/blob/development/.github/workflows/gh-pages.yml)
+- [creating tags and publishing to npm](https://github.com/NYPL/nypl-design-system/blob/development/.github/workflows/release.yml)
+
 ## Reporting results to Slack
 
 We use the [action-slack-notify](https://github.com/rtCamp/action-slack-notify) Github Action to post the results of GitHub actions to Slack. Here's an example configuration from the [circulation-patron-web](https://github.com/NYPL-Simplified/circulation-patron-web/) project, which posts a link to a report from an integration test suite:
```

Comment:
I'm not aware of this still working on any repos. I don't think its necessary as GHA will email people about failed jobs, so maybe we should replace this with a section on how to configure those notifications? (Unless Slack would be more useful as GH emails can be very noisy and tend to be ignored)